anybox/recipe/odoo/runtime/patch_odoo.py

```diff
@@ -12,7 +12,7 @@ def do_patch(gevent_script_path):
     isolated from the actual process management logic in the original.
     """
 
-    from openerp.service.server import PreforkServer, stripped_sys_argv
+    from odoo.service.server import PreforkServer, stripped_sys_argv
```

Comment:
Hi, 
Maybe my question is not relevant, but this change will break compatibility with previous version <10. Or did I miss something ?